react/features/face-centering/reducer.js

```diff
@@ -0,0 +1,51 @@
+import { ReducerRegistry } from '../base/redux';
+
+import {
+    START_FACE_RECOGNITION,
+    STOP_FACE_RECOGNITION,
+    UPDATE_FACE_COORDINATES
+} from './actionTypes';
+
+/**
+ * The default state object.
+ */
+const defaultState = {
+    /**
+     * Map of participant ids containing their respective facebox in the shape of a left, right, bottom, top percentages
+     * The percentages indicate the distance of the detected face starting edge (top or left) to the corresponding edge.
+     *
+     * Examples:
+     * 70% left indicates a 70% distance from the left edge of the video to the left edge of the detected face.
+     * 70% right indicates a 70% distance from the right edge of the video to the left edge of the detected face.
+     * 30% top indicates a 30% distance from the top edge of the video to the top edge of the detected face.
+     * 30% bottom indicates a 30% distance from the bottom edge of the video to the top edge of the detected face.
+     */
+    faceBoxes: {},
+    recognitionActive: false
```

Comment:
Could you please add a JS doc for this one as well?

I know it is obvious from the name but we try to put JS docs everywhere. So lets be consistent here as well.